README.md

```diff
@@ -245,27 +253,29 @@ console.log(datamodel.state); // {NetworkController: {...}, TokenRatesController
 console.log(datamodel.flatState); // {infura: {...}, contractExchangeRates: [...]}
 ```
 
-## Linking during development
+## Contributing
 
-Linking `@metamask/controllers` into other projects involves a special NPM command to ensure that dependencies are not duplicated. This is because `@metamask/controllers` ships modules that are transpiled but not bundled, and [NPM does not deduplicate](https://github.com/npm/npm/issues/7742) linked dependency trees.
+### Setup
 
-First, link `@metamask/controllers`.
+- Install [Node.js](https://nodejs.org) version 12
+  - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
+- Install [Yarn v1](https://yarnpkg.com/en/docs/install)
+- Run `yarn setup` to install dependencies and run any requried post-install scripts
+  - **Warning:** Do not use the `yarn` / `yarn install` command directly. Use `yarn setup` instead. The normal install command will skip required post-install scripts, leaving your development environment in an invalid state.
 
-```sh
-$ yarn build:link
-# or
-$ npm run build:link
-```
+### Testing and Linting
 
-Then, link into other projects.
+Run `yarn test` to run the tests once. To run tests on file changes, run `yarn test:watch`.
```

Comment:
There appears to be no `yarn test:watch` script in this repo
```suggestion
Run `yarn test` to run the tests.
```

Reply:
I added a watch script in 4782ed8